src/main/java/com/google/security/cryptauth/lib/securegcm/D2DConnectionContext.java

```diff
@@ -1,17 +1,17 @@
-/* Copyright 2018 Google LLC
- *
```

Comment:
It would be really nice if it were possible to make the java changes a separate PR.